src/akkudoktoreos/core/dataabc.py

```diff
@@ -1045,33 +1381,73 @@ def import_from_json(self, json_str: str, key_prefix: str = "") -> None:
             Given a JSON string with the following content:
             ```json
```

Comment:
The JSON example shows = instead of : for pairs. 
            ```json
            {
                "start_datetime": "2024-11-10 00:00:00",
                "interval": "30 minutes",
                "load_mean": [20.5, 21.0, 22.1],
                "other_xyz": [10.5, 11.0, 12.1]
            }
            ```

Reply:
Changed